assets/m_avas.json

```diff
@@ -1 +1,87 @@
+"045":{"keywords":"Cosmic","name":"Black Swan","points":"130"},
+"046":{"keywords":"Asgardian, Deity, Mystical","name":"Loki","points":"225/120"},
+"047":{"keywords":"Hellfire Club","name":"Master Pandemonium","points":"129"},
+"048":{"keywords":"Dark Avengers, Police, Thunderbolts","name":"Trick Shot","points":"104"},
+"049":{"keywords":"avengers, Invaders, S.H.I.E.L.D, Soldier","name":"Captain America","points":"150/75"},
+"050":{"keywords":"Armor, Avengers, Celebrity, Scientist, Stark Industries","name":"Iron Man","points":"242/125"},
+"051":{"keywords":"Animal, Avengers, Soldier, Warrior","name":"Red Wolf","points":"80"},
+"052":{"keywords":"Avengers, Martial Artist","name":"Ronan","points":"109"},
+"053":{"keywords":"Asgardian, Avengers, Deity, Warrior","name":"Thor","points":"228/100"},
+"053b":{"keywords":"Asgardian, Dark Avengers, Robot","name":"Ragnarok","points":"200"},
+"054":{"keywords":"Avengers, Cosmic","name":"Nightmask","points":"120"},
+"055":{"keywords":"Celebrity, Inhumans","name":"Ms. Marvel","points":"66"},
+"056":{"keywords":"Avengers, Builders, Cosmic","name":"Ex Nihilo","points":"190"},
+"057":{"keywords":"Initiative, Maggia, Mystical","name":"The Hood","points":"220/70"},
+"058":{"keywords":"Council of Kangs, Future, Past, Ruler","name":"Immortus","points":"160"},
+"059":{"keywords":"Badoon, Cosmic, Future, Robot, Scientist","name":"Korvac","points":"210"},
+"060":{"keywords":"HYDRA, Politician, Ruler, Scientist","name":"Red Skull","points":"200"},
+"061":{"keywords":"avegners, Nova Corps, Police","name":"Steve Rogers","points":"250"},
+"062":{"keywords":"Armor, Avengers, Scientist, Stark Industries","name":"Iron Man","points":"270"},
+"063":{"keywords":"Asgardian, Deity, Future, Ruler","name":"King Thor","points":"369/300/200"},
+"064":{"keywords":"Brute, Future, Monster, Ruler","name":"Hulk","points":"325/300"},
+"065":{"keywords":"Asgardian, Avengers, Deity, Spy","name":"Black Widow","points":"200"},
+"066":{"keywords":"Avengers","name":"Goliath","points":"98"},
+"067":{"keywords":"Avengers, Cosmic, Past","name":"Rick Jones","points":"75"},
+"Avid-001":{"keywords":"","name":"","points":"5"},
+"Avid-002":{"keywords":"","name":"","points":"5"},
+"Avid-003":{"keywords":"","name":"Falcon","points":"5"},
+"Avid-004":{"keywords":"","name":"","points":"5"},
+"Avid-005":{"keywords":"","name":"","points":"5"},
+"Avid-006":{"keywords":"","name":"Mockingbird","points":"5"},
+"Avid-007":{"keywords":"","name":"Nova","points":"5"},
+"Avid-008":{"keywords":"","name":"Ronin","points":"5"},
+"Avid-009":{"keywords":"","name":"Sandman","points":"5"},
+"Avid-010":{"keywords":"","name":"Sersi","points":"5"},
+"Avid-011":{"keywords":"","name":"","points":"5"},
+"Avid-012":{"keywords":"","name":"","points":"5"},
+"Avid-013":{"keywords":"","name":"","points":"5"},
+"Avid-014":{"keywords":"","name":"","points":"5"},
+"Avid-015":{"keywords":"","name":"Triathlon","points":"5"},
```

Comment:
That seems to be a constant mistake of mine.  I'll be sure to be more
careful in the future.  Thanks for the help and patience.
On May 7, 2015 8:21 AM, "Kyrre Havik Eriksen" <notifications@github.com>
wrote:

> In assets/m_avas.json
> <https://github.com/Kyrremann/ClixEmAll/pull/10#discussion_r29850382>:
>
> > +"067":{"keywords":"Avengers, Cosmic, Past","name":"Rick Jones","points":"75"},
> > +"Avid-001":{"keywords":"","name":"","points":"5"},
> > +"Avid-002":{"keywords":"","name":"","points":"5"},
> > +"Avid-003":{"keywords":"","name":"Falcon","points":"5"},
> > +"Avid-004":{"keywords":"","name":"","points":"5"},
> > +"Avid-005":{"keywords":"","name":"","points":"5"},
> > +"Avid-006":{"keywords":"","name":"Mockingbird","points":"5"},
> > +"Avid-007":{"keywords":"","name":"Nova","points":"5"},
> > +"Avid-008":{"keywords":"","name":"Ronin","points":"5"},
> > +"Avid-009":{"keywords":"","name":"Sandman","points":"5"},
> > +"Avid-010":{"keywords":"","name":"Sersi","points":"5"},
> > +"Avid-011":{"keywords":"","name":"","points":"5"},
> > +"Avid-012":{"keywords":"","name":"","points":"5"},
> > +"Avid-013":{"keywords":"","name":"","points":"5"},
> > +"Avid-014":{"keywords":"","name":"","points":"5"},
> > +"Avid-015":{"keywords":"","name":"Triathlon","points":"5"},
>
> The , :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kyrremann/ClixEmAll/pull/10/files#r29850382>.
>